config.py

```diff
@@ -1,5 +1,10 @@
 
 import os
 
-chromedriver_path = os.path.dirname(os.path.realpath(__file__)) + '/resources/chromedriver.exe'
+windows_chromedriver = 'chromedriver.exe'
 
+chromedriver_path = os.path.dirname(os.path.realpath(__file__)) + '/resources/' + windows_chromedriver
+
+
+homepage_url = 'https://www.coinbase.com/'
+signInPage_url = 'https://www.coinbase.com/signin'
```

Comment:
We don't want to put the full url of literally every page we automate. That would get huge. Only include the base url in the config and append the path in the page object.